OutOfSchool/OutOfSchool.WebApi/Services/Database/IWorkshopService.cs

```diff
@@ -145,4 +145,11 @@ Task<SearchResult<T>> GetByProviderId<T>(Guid id, ExcludeIdFilter filter)
     /// <param name="workshopId">WorkshopId for which we need to get provider owner entity.</param>
     /// <returns>A <see cref="Task"/> representing the result of the asynchronous operation.</returns>
     Task<Guid> GetWorkshopProviderOwnerIdAsync(Guid workshopId);
+
+    /// <summary>
+    ///  Returns isBlocked status of the workshop.
+    /// </summary>
+    /// <param name="workshopId">WorkshopId for which we need to get status.</param>
+    /// <returns>A <see cref="Task"/> representing the result of the asynchronous operation.</returns>
+    Task<bool> isBlocked(Guid workshopId);
```

Comment:
(Naming) Please use `PascalCase` while naming methods. Thus your `isBlocked` should be `IsBlocked` (starting with uppercase **I**)

Reply:
:) I should go back to the basics. Shame on me for that. :(
Fixed it.